_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,153 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul` , `ol` , and `dl` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
+
+## Expectation
+
+Each target element:
+
+- is categorized to follow the [flow content model][], and;
```

Comment:
I'm not sure what this means. `dl`, `ol` and `ul` elements are flow content. 
Can you come up with a Failed Example that would fail that item (and only that item)?

Reply:
Agree, this expectation can be removed.